### ISSUE TITLE
Status of project

### ISSUE THREAD

**Issue by EZarnosky** [RickDB/PlexMALSync#15]
I saw that MAL has a beta of their API, from what I can see it is dying in closed beta.

Any chance of reviving this project and using an unofficial API? [Jikan](https://jikan.moe/)  seems to have active development on [github](https://github.com/jikan-me/jikan) and even a [python](https://github.com/abhinavk99/jikanpy) verison.

I would also love a docker version of this much like you did for plexanisync

**Comment by kvloover** [comment]
"Notice: Jikan does not support authenticated requests. You can not update your lists."

**Comment by RickDB** [comment]
Spice needs an update or another proper (well documented) library for processing the MAL API has to appear before I can start working on it again sadly.

**Comment by allquiet-hub** [comment]
And now? There is this ["Update my anime list status"](https://myanimelist.net/apiconfig/references/api/v2#operation/anime_anime_id_my_list_status_put) which looks like what you need.

**Comment by RickDB** [comment]
Yeah it's, will wait a bit for libraries to catch up (preferably Spice).
API seems to be missing any search functions right now as well which we need for series lookups.

**Comment by jpf4e** [comment]
Spice seems to be dead.
Isn't there a search function?
https://myanimelist.net/apiconfig/references/api/v2#operation/anime_get

**Comment by RickDB** [comment]
Search is complete now.
Authentication seems to be missing documentation oddly enough:

https://myanimelist.net/apiconfig/references/api/v2#section/Authentication

Will need to dig up examples for that but this does look feasible, my time however is very limited and prefer AniList these days which has working integration with [PlexAniSync](https://github.com/RickDB/PlexAniSync).
If anyone wants to make a PR in the meantime you are welcome to do so :)
